buildSrc/src/main/java/com/google/firebase/gradle/plugins/ci/ChangedModulesTask.kt

```diff
@@ -43,14 +48,21 @@ abstract class ChangedModulesTask : DefaultTask() {
     val projects =
       AffectedProjectFinder(project, changedGitPaths.toSet(), listOf())
         .find()
+        .filter {
+          val ext = it.extensions.findByType(FirebaseLibraryExtension::class.java)
+          !onlyFirebaseSDKs || (ext != null && ext.publishJavadoc)
```

Comment:
I would remove the `publishJavadoc` check to make this task general purpose and not only useful for this workflow. iiuc it should not break anything since all firebase libraries have the `kotlindoc` task registered even if publishJavadoc is false